### ISSUE TITLE
IE doesn't set the subscriber's volume

### ISSUE THREAD

**Issue by ulifigueroa** [opentok/opentok-network-test#27]
Even here the volume is specified IE 11 doesn't change the volume and the user can her herself while doing the network test. https://github.com/opentok/opentok-network-test/blob/master/js-sample/app.js#L100

I also tried doing `subscriber.setAudioVolume(0);` and it doesn't' work? Is this a Tokbox plugin issue? Is there any workaround from the Javascript side?

**Comment by aullman** [comment]
Hm, can we use `subscriber.subscribeToAudio(false)` instead?

**Comment by davemun** [comment]
~`subscriber.subscribeToAudio(false)` in the `session.subscribe()` callback does work as a workaround at the moment.~

I have nevertheless filed a bug internally for the `setAudioVolume(0)` not working issue.

**Comment by ulifigueroa** [comment]
If I do that the network test will be running only for video. The drawback would be that the full bandwidth test (Audio and Video) cannot be done.

**Comment by davemun** [comment]
Ah, sorry, you are correct. I was just thinking about the volume part. `subscribeToAudio(false)` would not work for the original point of the test.

**Comment by ulifigueroa** [comment]
Yes, for muting the audio purposes I agree that's the correct solution but in this case it doesn't work. So right now users are hearing themselves during the system test, which is kinda awkward.
